plutus-ledger-api/src/Plutus/V1/Ledger/EvaluationContext.hs

```diff
@@ -36,29 +35,7 @@ import Data.Set as Set
 import Data.Text qualified as Text
 import GHC.Exts (inline)
 import GHC.Generics
-
-type DefaultMachineParameters =
-    Plutus.MachineParameters CekMachineCosts CekValue DefaultUni DefaultFun
-
--- | An opaque type that contains all the static parameters that the evaluator needs to evaluate a
--- script.  This is so that they can be computed once and cached, rather than recomputed on every
--- evaluation.
---
--- There are two sets of parameters: one is with immediate unlifting and the other one is with
--- deferred unlifting. We have to keep both of them, because depending on the language version
--- either one has to be used or the other. We also compile them separately due to all the inlining
--- and optimization that need to happen for things to be efficient.
-data EvaluationContext = EvaluationContext
-    { machineParametersImmediate :: DefaultMachineParameters
-    , machineParametersDeferred  :: DefaultMachineParameters
-    }
-    deriving stock Generic
-    deriving anyclass NFData
-
-toMachineParameters :: ProtocolVersion -> EvaluationContext -> DefaultMachineParameters
-toMachineParameters pv = case unliftingModeIn pv of
-    UnliftingImmediate -> machineParametersImmediate
-    UnliftingDeferred  -> machineParametersDeferred
+import Plutus.ApiCommon
 
 {- Note [Inlining meanings of builtins]
```

Comment:
move this note. Probably just delete this module tbh

Reply:
I was worried if ledger makes use of it, it is an exported module.

Reply:
https://github.com/input-output-hk/cardano-ledger/search?q=EvaluationContext

Reply:
ok I moved most to ApiCommon